specs/asgi.rst

```diff
@@ -306,26 +306,26 @@ There are times when protocol servers may want to provide server-specific
 extensions outside of a core ASGI protocol specification, or when a change
 to a specification is being trialled before being rolled in.
 
-For this use case, we define a common pattern for ``extensions`` - named
+For this use case, we define a common pattern for ``extensions``---named
 additions to a protocol specification that are optional but that, if provided
 by the server and understood by the application, can be used to get more
 functionality.
 
-This is achieved via a ``extensions`` entry in the ``scope`` dict, which is
-itself a dict. Extensions have a unicode string name that
+This is achieved via an ``extensions`` entry in the ``scope`` dictionary, which
+is itself a ``dict``. Extensions have a Unicode string name that
 is agreed upon between servers and applications.
 
 If the server supports an extension, it should place an entry into the
-``extensions`` dict under the extension's name, and the value of that entry
-should itself be a dict. Servers can provide any extra scope information
-that is part of the extension inside this dict value, or if the extension is
-only to indicate that the server accepts additional events via the ``send``
-callable, it may just be an empty dict.
+``extensions`` dictionary under the extension's name, and the value of that
+entry should itself be a ``dict``. Servers can provide any extra scope
+information that is part of the extension inside this value or, if the
+extension is only to indicate that the server accepts additional events via
+the ``send`` callable, it may just be an empty ``dict``.
 
-As an example, imagine a HTTP protocol server wishes to provide an extension
+As an example, imagine an HTTP protocol server wishes to provide an extension
```

Comment:
I would not choose to put `an` before a strong `H` sound, like in HTTP.

Reply:
Ah, this may be a British/American English difference I wasn't familiar with 😉I tried to avoid those on the whole, happy to back this out!

Reply:
Yes, it's probably the *aitch* versus *haitch* pronunciation thing. While Django doc standard is US English, I would like to leave it as `a`, just for me :)